releasenotes/notes/0.24/trotter-time-dep-hamiltonians-8c6c3d5f629e72fb.yaml

```
@@ -1,8 +1,18 @@
 ---
-features:
+features_algorithms:
   - |
-    Add support for time-dependent Hamiltonians in 
-    :class:`~.time_evolvers.TrotterQRTE`.
-  - | 
-    Add support for observable evaluations at every time-step in 
-    :class:`~.time_evolvers.TrotterQRTE`.
+    Added support for handling time-dependent Hamiltonians (i.e. singly
+    parametrized operators) to the
+    :class:`~qiskit.algorithms.time_evolvers.trotterization.TrotterQRTE` class.
+    To facilitate working with this, added the
+    :attr:`~qiskit.algorithms.time_evolvers.trotterization.TrotterQRTE.num_timestamps`
```

Stamps would also be an option, but we chose steps 😛 
```suggestion
    :attr:`~qiskit.algorithms.time_evolvers.trotterization.TrotterQRTE.num_timesteps`
```